create_jenkins_job.py

```diff
@@ -121,7 +121,7 @@ def main(argv=None):
         'dont_notify_every_unstable_build': 'false',
         'build_timeout_mins': 0,
         'ubuntu_distro': 'jammy',
-        'el_release': '9',
+        'el_release': '9.1',
```

Comment:
I'll point out that this is not a long-term solution; people won't be able to install ROS 2 on RHEL 9.2 or later.  However, I think for now it is better to be running CI on RHEL (even on an old version), than not to be running at all.  So I'm OK with the change.

Because this is a change to `create_jenkins_job.py`, this won't have an effect until it is deployed.  I'll suggest deploying this to only the `test_` jobs (something like `create_jenkins_job.py --select-jobs-regexp 'test_*'`), and then running a new CI job using https://ci.ros2.org/view/All/job/test_ci_linux-rhel/.  Once we've determined that this works fine for the test job, we can then merge it and deploy it to everything.

Reply:
Agree that this is not a long term solution. 
I will do the deployment on the test jobs and we can check if this solves it.
Long term in these scenarios we would either need to not depend on EPEL and upstream rebuild to keep supporting newer versions of RHEL, otherwise we are bound to the cadence of rebuilts on EPEL.